cloud_infrastructure_access/rosa-aws-privatelink.adoc

```diff
@@ -0,0 +1,17 @@
+[id="rosa-aws-privatelink"]
+= Configuring AWS PrivateLink
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-aws-privatelink
+
+toc::[]
+
+include::modules/rosa-configure-privatelink-about.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-architecture.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-requirements.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-configuration.adoc[leveloffset=+1]
+
+== Additional resources
+
+* See xref:../rosa_getting_started/rosa-setting-up-environment.adoc[Setting up the environment] to prepare for installing {product-title} (ROSA).
+* For an overview of installing into an existing VPC, see link:https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your VPC] in the OCP documentation.
```

Comment:
s/installing into an existing VPC/installing ROSA using an existing VPC

And also, check with Will Gordon to see if the better link to point them to is this one in the ROSA docs: https://docs.openshift.com/rosa/cloud_infrastructure_access/dedicated-aws-peering.html

instead of pointing them to the OCP docs. I believe that they should be looking at the ROSA topic instead.

Reply:
@wgordon17    Could you respond to JQ's question on whether the link to the OCP topic:

Requirements for using your VPC
https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc

Should be replaced with a link to this topic in the ROSA docs:

Configuring AWS VPC peering
https://docs.openshift.com/rosa/cloud_infrastructure_access/dedicated-aws-peering.html

Reply:
_Ideally_, we would have a better, more prescripted set of directions on this. I'm hoping to improve this on the OCP with https://issues.redhat.com/browse/OSDOCS-1968, but in the meantime, if all we have is the OCP docs, then that's all we can point to. The VPC peering doc is not the right content that we want to point to.